routes/v1/completed_tasks/completed_tasks.go

```diff
@@ -41,42 +67,73 @@ func DeleteCompletedTaskByCompletedTaskId(w http.ResponseWriter, r *http.Request
 	vars := mux.Vars(r)
 	utils.EnableCors(&w)
 
-	paramsCompletedTaskId := vars["completedTaskId"]
-
-	oid, err := primitive.ObjectIDFromHex(paramsCompletedTaskId)
+	if r.Method == "DELETE" {
+		paramsCompletedTaskId := vars["completedTaskId"]
+		paramsFireBaseUserId := vars["fireBaseUserId"]
 
-	if err != nil {
-		log.Println("Error converting params completedTaskId to ObjectId:", err)
-		w.Header().Set("Content-Type", "application/json")
-		var iceCreamError errors_models.IceCreamErrors
-		iceCreamError.Error = err.Error()
-		iceCreamError.Info = "Invalid completedTaskId provided"
-		json.NewEncoder(w).Encode(iceCreamError)
-	} else {
-		res, err := completed_tasks_controllers.DeleteCompletedTaskByCompletedTaskId(oid)
+		oid, err := primitive.ObjectIDFromHex(paramsCompletedTaskId)
 
 		if err != nil {
-			log.Println("Error deleting completedTask:", err)
+			log.Println("Error converting params completedTaskId to ObjectId:", err)
 			w.Header().Set("Content-Type", "application/json")
 			var iceCreamError errors_models.IceCreamErrors
 			iceCreamError.Error = err.Error()
-			iceCreamError.Info = "Error deleting completedTask"
+			iceCreamError.Info = "Invalid completedTaskId provided"
 			json.NewEncoder(w).Encode(iceCreamError)
-		}
-
-		if res.DeletedCount == 1 {
-			var deleteResult utils_models.DeleteResult
-			deleteResult.Info = "Successfully deleted Completed Task"
-			deleteResult.Success = true
-			w.Header().Set("Content-Type", "application/json")
-			json.NewEncoder(w).Encode(deleteResult)
 		} else {
-			log.Println("could not find matching completedTask ID:", oid)
-			w.Header().Set("Content-Type", "application/json")
-			var iceCreamError errors_models.IceCreamErrors
-			iceCreamError.Error = fmt.Sprintf("could not find matching completedTask ID: %s", oid)
-			iceCreamError.Info = "Error deleting rule: no matching oid"
-			json.NewEncoder(w).Encode(iceCreamError)
+			res, err := completed_tasks_controllers.DeleteCompletedTaskByCompletedTaskId(oid)
+
+			if err != nil {
+				log.Println("Error deleting completedTask:", err)
+				w.Header().Set("Content-Type", "application/json")
+				var iceCreamError errors_models.IceCreamErrors
+				iceCreamError.Error = err.Error()
+				iceCreamError.Info = "Error deleting completedTask"
+				json.NewEncoder(w).Encode(iceCreamError)
+			}
+
+			if res.DeletedCount == 1 {
+				// var deleteResult utils_models.DeleteResult
```

Comment:
I think it's better to delete those comment out.(line 96-98, 127-128)

Reply:
Thanks, removed the commented out code!